projects/1.18-fabric/assets/incantationem/incantationem/lang/zh_cn.json

```
@@ -0,0 +1,43 @@
+{
+  "enchantment.incantationem.bunnys_hop": "兔跳",
+  "enchantment.incantationem.swine_bane": "猪类灾祸",
+  "enchantment.incantationem.charmed": "庇佑",
+  "enchantment.incantationem.decay": "破败",
+  "enchantment.incantationem.forging_touch": "煅烧之触",
+  "enchantment.incantationem.last_stand": "背水一战",
+  "enchantment.incantationem.magnetic": "磁性",
+  "enchantment.incantationem.reaping_rod": "收割之竿",
+  "enchantment.incantationem.retreat": "撤退",
+  "enchantment.incantationem.venomous": "淬毒",
+
+  "enchantment.incantationem.recklessness": "鲁莽诅咒",
+  "enchantment.incantationem.thunder": "引雷诅咒",
+  "enchantment.incantationem.tough_luck": "不幸诅咒",
+
+
+  "message.incantationem.decay.applied": "*已施加破败效果！*",
+  "message.incantationem.forging_touch.applied": "*已施加煅烧之触效果！*",
+  "message.incantationem.last_stand.applied": "*已施加背水一战效果！*",
+  "message.incantationem.reaping_rod.applied": "*已施加收割之竿效果！*",
+  "message.incantationem.retreat.applied": "*已施加撤退效果！*",
+  "message.incantationem.venomous.applied": "*已施加淬毒效果！*",
+
+  "message.incantationem.thunder.applied": "*引雷诅咒出现了！：（*",
+
+  "enchantment.incantationem.bunnys_hop.desc": "站在泥土/草方块类方块上提供跳跃提升效果。",
+  "enchantment.incantationem.swine_bane.desc": "提高对猪类生物的伤害，例如猪，猪灵，疣猪兽等。",
+  "enchantment.incantationem.charmed.desc": "提供幸运效果，并且提高触发本模组增益效果的几率，降低本模组诅咒触发的几率。",
+  "enchantment.incantationem.decay.desc": "攻击有一定概率造成凋零效果。",
+  "enchantment.incantationem.forging_touch.desc": "破坏方块有一定几率掉落其烧制后的产物。（比如：沙子→玻璃）",

```

```suggestion
  "enchantment.incantationem.forging_touch.desc": "破坏方块有一定概率掉落其烧制后的产物。（比如：沙子->玻璃）",
```
```suggestion
  "enchantment.incantationem.forging_touch.desc": "破坏方块有一定概率掉落其烧制后的产物。（比如：沙子→玻璃）",
```
→符号的修改仅是为mc自带字体考虑，若非强制Unicode模式下→符号与英文字母粗细显示一致可不改
这个修改完全不强制